pkg/builder/build.go

```diff
@@ -52,6 +54,11 @@ func Build(options Options, dirs ...string) (string, error) {
 	}
 
 	StepLogger.Printf("Pushing image %s (insecure=%v)...", options.Target, options.PushInsecure)
+
+	if options.jobs == 0 {
```

Comment:
I'd be inclined not to enforce any default at Spectrum level, so callers can still leverage the go-containerregistry defaulting if they want, and only pass the option when it's been set.